packages/language-services/src/features/do-complete.ts

```diff
@@ -256,6 +280,7 @@ export class DoComplete extends LanguageFeature {
 			const documents = this.cache.documents();
 			for (const currentDocument of documents) {
 				if (
+					document.languageId === "scss" &&
```

Comment:
We might not need this check here when we call upstream later anyway. Duplicates?